jobserver/management/commands/inflate_release_files.py

```diff
@@ -0,0 +1,129 @@
+import csv
+import json
+
+from django.core.management.base import BaseCommand
+
+from ...models import ReleaseFile
+
+
```

Comment:
suggestion: Maybe a module-level docstring is worth adding to explain what this does? Or some explanation in the developer docs somewhere?